csharp/src/Google.Protobuf/CodedInputStream.cs

```diff
@@ -424,7 +424,10 @@ public void SkipLastField()
             }
         }
 
-        private void SkipGroup(uint startGroupTag)
+        /// <summary>
+        /// Skip a group.
+        /// </summary>
+        public void SkipGroup(uint startGroupTag)

```

Comment:
Does this need to be public now? We're skipping it within the library, aren't we? Maybe just internal?

Reply:
Done